data/json/items/containers.json

```diff
@@ -14,13 +14,22 @@
     "material": "plastic",
     "symbol": ")",
     "color": "dark_gray",
+    "//guinea pig": "fill out all the pocket properties to wrap my head around them, hence the excessive comments ",
     "pocket_data": [
       {
         "pocket_type": "CONTAINER",
+        "min_item_volume": "1 ml",
```

Comment:
this isn't really that helpful to add, and imo it just clogs up the iteminfo screen for players.